wallet/udb/upgrades.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2017 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package udb
+
+import (
+	"github.com/decred/dcrwallet/apperrors"
+	"github.com/decred/dcrwallet/walletdb"
+)
+
+const (
+	initialVersion = iota + 1
```

Comment:
Understood why you'd prefer this (to make it easier to see exactly what version some constant is by just reading the code).  I guess i'm spoiled since I can just mouse over it and see what the constant really is.